### PR TITLE
Nav Bar Resolves #16

### LINKED ISSUE #16: Nav Component
Will build out nav component

### PR DESCRIPTION
**Pull request by Josh-Kaiser-OSU**
Resolves #16 
Makes Nav component.
On Profile page it shows "Browse" button on the right part of the nav bar. On every other page it shows the "Profile" button.

Screen Shot 2021-10-16 at 6 04 18 PM

Screen Shot 2021-10-16 at 6 04 07 PM

### CODE REVIEW

**Comment by Josh-Kaiser-OSU** [comment]
I forgot to say, it also creates the skeleton for a myProfile page where the singed-in users profile will be displayed.

It also adds an addClassName property to the Button component if we want to add additional styles to an individual Button.

File: portfolio-project-matching-app/components/Button.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import styles from './Button.module.css';
 
-const Button = ({text, type='btn', onClick}) => {
+const Button = ({text, type='btnGeneral', onClick, addClassName=""}) => {
```
Comment (kiserd) [inline portfolio-project-matching-app/components/Button.js:3]: Super smart idea. No matter how much we try to stick to the cookie cutter component, it  always helps to be able to inject some context-specific styling. Love it.

File: portfolio-project-matching-app/components/Nav.js [hunk 2]
```diff
@@ -1,19 +1,42 @@
...
+                            </a>
+                        </Link>
+                        :
+                        <Link href="/myProfile" passHref>
+                            <a>
+                            <Button text="Profile" isLink={true} addClassName="bg-white"/>
+                            </a>
+                        </Link>
+
+                    }
```
Comment (kiserd) [inline portfolio-project-matching-app/components/Nav.js:39]: Not sure I've had the opportunity to use `useRouter`, thanks for demonstrating it here. I was able to clearly see how you used the path variable to determine what should be displayed.

File: portfolio-project-matching-app/pages/myProfile.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const myProfile = () => {
+    return(
+        <div>
+
+        </div>
+    )
+}
+
+export default myProfile;
```
Comment (kiserd) [inline portfolio-project-matching-app/pages/myProfile.js:9]: Nice! We officially have a page!

**Review by kiserd — APPROVED** [review]
I think everything looks grand. I'm sure we will tinker with the colors a bit once we've decided what we want our "main" color to be. My gut was thinking the nav bar ought to have a dark background with white text, but that is probably just what I'm used to seeing.

Awesome job, Josh!

**Pull request merged**